rdt/transformers/numerical.py

```diff
@@ -487,3 +490,204 @@ def _reverse_transform(self, data):
             data = self._univariate.ppf(scipy.stats.norm.cdf(data))
 
         return super()._reverse_transform(data)
+
+
+class BayesGMMTransformer(NumericalTransformer):
+    """Transformer for numerical data using a Bayesian Gaussian Mixture Model.
+
+    The transformation consists of computing the probability that a value came from each
+    component, then sampling a component based on that probability and finally returning a
+    pd.DataFrame of a value and a label encoding, where the lable encoding indicates the
```

Comment:
this description seems to be a bit off, since there is no label encoding and the result of the transformation is not a DataFrame  (the output of the `transform` is, but I'd rather talk about columns instead).